suite/tests/client-interface/cleancall-opt-shared.h

```diff
@@ -518,6 +544,7 @@ before_callee(app_pc func, const char *func_name)
                           DR_MEMPROT_EXEC|DR_MEMPROT_READ|DR_MEMPROT_WRITE);
     }
 
+#ifdef TEST_INLINE
     ilist = instrlist_create(dc);
 #ifdef X86
```

Comment:
nit, we should adjust the indentation, also, the comment on #endif about which macro it ends would be helpful.